misc/bash_completion.d/tarsnap

```diff
@@ -59,7 +59,8 @@ _tarsnap ()
 		  --no-snaptime --no-store-atime --no-totals --store-atime --totals --recover \
 		  --insane-filesystems --quiet --no-quiet --configfile --no-default-config --fsck-prune \
 		  --retry-forever --no-retry-forever --creationtime \
-		  --keep-going --verify-config --initialize-cachedir"
+		  --keep-going --verify-config --initialize-cachedir \
+		  --iso-dates"
```

Comment:
We need a --no-iso-dates option too, in case someone specified `iso-dates` in a configuration file.